third_party/protobuf/3.6.1/protobuf.bzl

```diff
@@ -266,7 +266,7 @@ def internal_gen_well_known_protos_java(srcs):
   Args:
     srcs: the well known protos
   """
-  root = Label("%s//protobuf_java" % (REPOSITORY_NAME)).workspace_root
+  root = Label("%s//protobuf_java" % repository_name()).workspace_root
```

Comment:
this needs to be native.repository_name()

Reply:
Fixed